packages/web-ui/src/providers/SessionProvider/index.tsx

```diff
@@ -7,10 +7,9 @@ export type SessionUser = {
   name: string | null | undefined
   email: string
 }
-export type SessionWorkspace = { id: number; name: string }
+
 interface ISessionContext {
   currentUser: SessionUser
-  workspace: SessionWorkspace
```

Comment:
Ok, this is related to my other comment. I would like to understand this change

Reply:
This is related to the other change I explained above. I deleted the workspace from the useSession, and replace it with the useCurrentWorkspace hook (store?), so that the workspace value actually changes when updated.